newIDE/app/src/ProjectManager/index.js

```diff
@@ -576,7 +576,7 @@ export default class ProjectManager extends React.Component<Props, State> {
                 src="res/ribbon_default/projectManager32.png"
               />
             }
-            initiallyOpen={false}
+            initiallyOpen={true}
```

Comment:
Don't you think that would confuse new users? From a few sessions I observed, some people new to the software are not even sure where they should click after opening a game with more than one scene, so if they have a bunch of settings that they don't care about showing on the top, might not be the best place. Maybe we should move it to the bottom.

Reply:
I think that as it is we just add an extra click to reveal  the first things people will need to do when starting a project- set the project :) But I will revert it usage data shows that some find seeing them overwhelming by default

Reply:
Mmmm ok... the first impression is very important and I'm afraid that everything should be made to make as smooth as possible for brand new users. Again, I've seen people being confused what to do after opening a game before I added the automatic opening of the first scene (in case there is only one scene).

Reply:
ok, reverted it :)